src/app/ubs/ubs/components/ubs-order-details/ubs-order-details.component.html

```diff
@@ -34,15 +34,16 @@ <h5>{{ 'order-details.current-location' | translate: { currentLocation: currentL
             <div class="col-2 form-group count">
               <input
                 formControlName="quantity{{ bag.id }}"
-                type="number"
                 class="shadow-none form-control input"
+                type="number"
                 [id]="'quantity' + bag.id"
                 min="0"
                 max="999"
                 placeholder="0"
                 [unmask]="false"
                 [imask]="{ mask: servicesMask }"
                 (change)="onQuantityChange(bag.id)"
+                (keypress)="($event.charCode >= 48 && $event.charCode < 58)"
```

Comment:
probably, ($event.charCode >= 48 && $event.charCode <= 57) || ($event.charCode >= 96 && $event.charCode <= 105), but charCode is deprecated now. Use $event.key here:

const key = Number(e.key)
if (isNaN(key) || e.key === null || e.key === ' ') {
  // not numeric
}
else {
  // numeric
}